libraries/ios/Demo App/SceneDelegate.swift

```diff
@@ -17,8 +17,19 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         let window = UIWindow(windowScene: windowScene)
 
         /// 3. Create a view hierarchy programmatically
+        ///
+        /// By default, we'll use a pre-compiled bundle that's embedded in the demo app
         let viewController = WCReactNativeViewController()
 
+        /// But you might want to reference a running metro server (which can be used from a simulator):
+        /// let viewController = WCReactNativeViewController.forLocalDevelopment
```

Comment:
```suggestion
        /// let viewController = WCReactNativeViewController.forLocalDevelopment()
```

Minor, but it wasn't obvious to me at the beginning what's missing after uncommenting this line.

Reply:
Ah yep thanks!!